backend/capellacollab/settings/configuration/models.py

```diff
@@ -37,6 +38,62 @@ class MetadataConfiguration(core_pydantic.BaseModelStrict):
     environment: str = pydantic.Field(default="-", description="general")
 
 
+class BuiltInLinkItem(str, enum.Enum):
+    GRAFANA = "grafana"
+    PROMETHEUS = "prometheus"
+    DOCUMENTATION = "documentation"
+
+
+class UserRole(str, enum.Enum):
```

Comment:
We already have a class Role in `users/models.py`, which we can make use of. Don't declare it another time.